typescript/scripts/sort-msw-index.ts

```diff
@@ -0,0 +1,14 @@
+import fs from "node:fs";
```

Comment:
This script will deterministically format the `index.msw.ts` file, which currently does not get deterministically formatted and would cause false positives